CSS Issue

sharmamohit123 opened the pull request:
Signed-off-by: Mohit Sharma <mohitdonsharma9@gmail.com>

### Description
Fixed position issue
Please describe your pull request.

Fixes https://github.com/phpmyadmin/phpmyadmin/issues/14133

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

sharmamohit123 committed:
CSS Issue

Signed-off-by: sharmamohit123 <mohitdonsharma9@gmail.com>

sharmamohit123 committed:
CSS Issue

Signed-off-by: Mohit Sharma <mohitdonsharma9@gmail.com>

shucon commented:
@mauriciofauth Looks good to me.

MauricioFauth committed:
Add Yarn cache and install yarn on osx for TravisCI

Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

Commit:
Clarify a message (#14628)

Signed-off-by: Marc Delisle <marc@infomarc.info>

MauricioFauth committed:
Update po files

[CI skip]

Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

Commit:
Translated using Weblate (Slovenian)

Currently translated at 100.0% (3298 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/Development
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/master/sl/
Signed-off-by: Domen <mitenem@outlook.com>

MauricioFauth committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (3298 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/Development
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/master/pt_BR/
Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (3298 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/Development
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/master/fr/
Signed-off-by: Marc Delisle <marc@infomarc.info>

comradekingu committed:
Translated using Weblate (Norwegian Bokmål)

Currently translated at 56.9% (1878 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/Development
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/master/nb/
Signed-off-by: Allan Nordhøy <epost@anotheragency.no>

BouRock committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (3298 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/Development
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/master/tr/
Signed-off-by: Burak Yavuz <hitowerdigit@hotmail.com>

MauricioFauth committed:
Fix PHP notice

Undefined index: git_location

Signed-off-by: Maurício Meneghini Fauth <mauriciofauth@gmail.com>

weblate committed:
Merge branch 'origin/master' into Weblate

Commit:
Translated using Weblate (Ukrainian)

Currently translated at 100.0% (3298 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/Development
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/master/uk/
Signed-off-by: Володимир Бриняк <bardvv@gmail.com>

victorffarias committed:
Issue #12262 - Removing inline styles (#14638)

Signed-off-by: Victor Farias <victor.f.farias@gmail.com>

weblate committed:
Merge branch 'origin/master' into Weblate

mauriciofauth commented:
Hello. Thanks for your contribution!

Could you use your full name in the signature tag?
For example: `Signed-off-by: Jane Developer <jane@example.org>`

See [Developer's Certificate of Origin](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO#L39)

SantosSi committed:
Translated using Weblate (German)

Currently translated at 100.0% (3298 of 3298 strings)

[CI skip]

Translation: phpMyAdmin/Development
Translate-URL: https://hosted.weblate.org/projects/phpmyadmin/master/de/
Signed-off-by: ssantos <ssantos@web.de>

sharmamohit123 committed:
Merge branch 'designer' of https://github.com/sharmamohit123/phpmyadmin into designer

sharmamohit123 commented:
Creating new PR
